ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml

```diff
@@ -162,6 +162,18 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       TEST_ARGS: --from-repository quay.io/multi-arch/community-e2e-images
     workflow: openshift-e2e-aws-image-ecosystem
+- as: ocp-jenkins-e2e-aws-arm64
+  interval: 24h
+  steps:
+    cluster_profile: aws-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: arm-ci.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      TEST_ARGS: --from-repository quay.io/multi-arch/community-e2e-images
+      TEST_TYPE: openshift/jenkins-e2e
```

Comment:
iiuc this should be `TEST_TYPE: jenkins-e2e-rhel-only` but still not sure that we should add this at this point per my previous comment